src/coupled/ESM1.5/CABLEfilesFromESM1.5/cable_rad_driver.F90

```diff
@@ -55,7 +55,8 @@ SUBROUTINE cable_rad_driver(                                                   &
    USE cable_common_module, ONLY : cable_runtime, cable_user
    
 USE cbl_init_radiation_module, ONLY: Common_InitRad_Scalings
-USE grid_constants_mod_cbl, ONLY : ICE_SoilType, lakes_cable
+USE grid_constants_mod_cbl,  ONLY: ICE_SoilType
```

Comment:
Indentation.

Reply:
hmm - I dont get the comment. The indentation level is the same as it was and matches the existing lines around it?